### DISCUSSION TITLE
Is there any way to disable scroll position recovery ?

### DISCUSSION THREAD

**Discussion by berekashvili22** [vercel/swr#2176] [Q&A]
When using useSWR hook page scrolls top automatically , i am wondering if there is any way to disable that behavior ?
I had same problem with regular fetch but solved it by adding `history.scrollRestoration = 'manual';`
P.S I replaced useSWR with regular fetch and everything worked fine

**Comment by feledori** [comment]
Hmmm not really sure if this has something to do with SWR or with the configuration of your router. If you are using SWR for pagination maybe this can help [SWR Pagination](https://swr.vercel.app/docs/pagination).
